ES6/01 no-treeshaking-material-ui/readme.md

```diff
@@ -0,0 +1,94 @@
+# No tree shaking material ui
+
+This sample start from [00 Base](../00%20base/), in this case we are going to install a popular ui library (material ui) and we are going to import components from there in way that tree shaking is not enabled.
+
+# Steps
+
+Steps to recreate this sample:
+
+- Copy the content from _[00 Base](../00%20base/)_
+
+- First install the packages needed, run from the bash / cmd:
+
+```bash
+npm install
+```
+
+- Now let's install _[material_ui](https://www.material-ui.com/#/)_ library.
+
+```
+npm install material-ui --save
+```
+
+- Let's install the typings for this library:
+
+```
+npm install @types/material-ui --save-dev
+```
+
+- Let's add a simple component (in a way that three shaking is not triggered):
+
+_[app.jsx](./src/app.jsx)_
+```diff
+import * as React from 'react';
+import { HelloComponent } from './hello';
+import { NameEditComponent } from './nameEdit';
++ import {AppBar} from "material-ui";
+
+export class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      userName: 'defaultUserName',
+    };
+
+    this.setUsernameState = this.setUsernameState.bind(this);
+  }
+
+  setUsernameState(event) {
+    // If the state gets more complex we should use object.assign
```

Comment:
Remove this comment (is my fault, from my code base), now makes no sense, thanks

Reply:
I already eliminated the complete demo 1, it was assigned to another partner.